addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -20,13 +30,21 @@
         <span t-else="" class="fw-bold text-uppercase">Free</span>
     </xpath>
     <xpath expr="//div[hasclass('o_wevent_price_range')]" position="inside">
-        <t t-if="event.event_ticket_ids[-1].price_reduce > 0">
+        <t t-if="event.event_ticket_ids[0].price_reduce &lt;= event.event_ticket_ids[-1].price_reduce">
```

Comment:
small bug here, it work well if you have only 2 tickets, but if you have 3 the min / max might be wrong (if one if the extreme value is at the middle)

![screenshot_spect](https://user-images.githubusercontent.com/54316226/211499826-3577a0d7-23fb-48fd-90c1-e4074e5b1794.png)

To fix it, you can do something like
```xml
<t t-set="all_prices" t-value="event.event_ticket_ids.mapped('price_reduce')"/>
<t t-set="lowest_price" t-value="min(all_prices)"/>
<t t-set="highest_price" t-value="max(all_prices)"/>
```